### PR TITLE
Location ui

### PR DESCRIPTION
**Pull request by emord**
@esoergel 

Displays location settings for web users whenever locations are enabled without commtrack and restriction checkbox

@snopoke

### CODE REVIEW

**Comment by snopoke** [comment]
Kicked the build

File: corehq/apps/users/templates/users/edit_web_user.html [hunk 1]
```diff
@@ -60,7 +60,11 @@
         <form id="commtrack_form" class="form form-horizontal" name="" method="post">
             <input type="hidden" name="form_type" value="commtrack" />
             <fieldset>
-                <legend>{% trans 'CommTrack Settings' %}</legend>
+                {% if commtrack_enabled %}
+                    <legend>{% trans 'CommTrack Settings' %}</legend>
+                {% else %}
+                    <legend>{% trans 'Location Settings' %}</legend>
+                {% endif %}
```
Comment (esoergel) [inline corehq/apps/users/templates/users/edit_web_user.html]: :+1:

File: corehq/feature_previews.py [hunk 2]
```diff
@@ -143,7 +143,7 @@ def enable_locations(domain_name, checked):
     description=_(
         'Enable locations for this project. This must be enabled for CommTrack to work properly'
     ),
-    help_link='http://help.commcarehq.org/',
```
Comment (esoergel) [inline corehq/feature_previews.py]: wait, was that really the link?? That's preposterous.

> For more information, check out our [documentation](http://lmgtfy.com/?q=go+fuck+yourself)

Comment (czue) [inline corehq/feature_previews.py]: lol

**Pull request merged**